sdk/src/main/java/com/mapbox/maps/renderer/widget/Widget.kt

```diff
@@ -6,7 +6,7 @@ import com.mapbox.maps.MapboxExperimental
  * Base class for widgets displayed on top of the map.
  */
 @MapboxExperimental
-abstract class Widget {
+abstract class Widget internal constructor() {
```

Comment:
perhaps I don't see full context but I'd actually go with exposing `Widget` fully. Would be glad if more details could be provided in PR description why do we decide to mark this abstract class as internal